server/routes/app.js

```diff
@@ -30,7 +30,8 @@ router.get(
         googleAuthConfigured: config.googleAuthConfigured(),
         localAuthConfigured: !config.get('disableUserpassAuth'),
         publicUrl: config.get('publicUrl'),
-        samlConfigured: Boolean(config.get('samlEntryPoint')),
+        samlConfig:
```

Comment:
Could we leave in `samlConfigured` and send down `samlLinkHtml` as a separate, optional config field? I don't mind supporting custom link HTML, but I think it shouldn't be required. The default button today could be changed to something like "Sign in with SSO" or something generic to fix the ADFS  naming